ecosystem/typescript/sdk/src/aptos_account.ts

```diff
@@ -1,13 +1,15 @@
 import * as Nacl from "tweetnacl";
 import * as SHA3 from "js-sha3";
+import * as bip39 from "@scure/bip39";
 import { Buffer } from "buffer/"; // the trailing slash is important!
+import { derivePath } from "ed25519-hd-key";
 import { HexString, MaybeHexString } from "./hex_string";
-import * as Gen from "./generated/index";
+import { Types } from "./types";
```

Comment:
SDK has been refactored.. We should keep `import * as Gen from "./generated/index";`.